test/phone.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config({
+    path: require('path').join(__dirname, '../.env')
+});
+
+var assert = require('assert');
+var app = require('../cyc_entry.js');
+var request = require('request');
+var server;
+var PORT = 8082;
+var URL = 'http://localhost:' + PORT;
+
+describe('phone', function () {
+    beforeEach(function (done) {
+        server = app.listen(PORT, done);
+    });
+
+    afterEach(function (done) {
+        server.close(done);
+    });
+
+    it('new call', function (done) {
+        request.post(URL + '/new_phone_call', function (err, res, body) {
+            if (err) return done(err);
+            assert.equal(res.statusCode, 200);
+            assert.notEqual(
+                body.indexOf('audio/v2/zip_prompt.mp3'), -1,
+                '/new_phone_call should play audio/v2/zip_prompt.mp3');
+            done();
+        });
+    });
+
+    describe('redirect call', function () {
+        it('enforces zip code', function (done) {
+            // Todo: Add a test for auto-finding the zip code of a repeat caller
```

Comment:
I usually find it best to mark the owner of each TODO, like `// TODO(thosakwe): Add a test for auto-finding...`.  This is useful for people reading the code and looking to help with todos, and also improves accountability in the codebase.